[2021-03-06 17:59:04,476] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: nhunh_example.zenodo_downloader 2021-03-06T17:58:15.355590+00:00 [queued]>
[2021-03-06 17:59:04,494] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: nhunh_example.zenodo_downloader 2021-03-06T17:58:15.355590+00:00 [queued]>
[2021-03-06 17:59:04,494] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-06 17:59:04,494] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-03-06 17:59:04,494] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-06 17:59:04,506] {taskinstance.py:1063} INFO - Executing <Task(ZenodoDownloaderOperator): zenodo_downloader> on 2021-03-06T17:58:15.355590+00:00
[2021-03-06 17:59:04,510] {standard_task_runner.py:52} INFO - Started process 43 to run task
[2021-03-06 17:59:04,513] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'nhunh_example', 'zenodo_downloader', '2021-03-06T17:58:15.355590+00:00', '--job-id', '2', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/airflow_example.py', '--cfg-path', '/tmp/tmpxbg0txpz', '--error-file', '/tmp/tmpc78nrd0r']
[2021-03-06 17:59:04,513] {standard_task_runner.py:77} INFO - Job 2: Subtask zenodo_downloader
[2021-03-06 17:59:04,561] {logging_mixin.py:104} INFO - Running <TaskInstance: nhunh_example.zenodo_downloader 2021-03-06T17:58:15.355590+00:00 [running]> on host 2ca80f9c51df
[2021-03-06 17:59:04,675] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=nunovazafonso
AIRFLOW_CTX_DAG_ID=nhunh_example
AIRFLOW_CTX_TASK_ID=zenodo_downloader
AIRFLOW_CTX_EXECUTION_DATE=2021-03-06T17:58:15.355590+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-06T17:58:15.355590+00:00
[2021-03-06 17:59:04,676] {data_downloader.py:41} INFO - Zenodo Downloader
[2021-03-06 17:59:04,685] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=nhunh_example, task_id=zenodo_downloader, execution_date=20210306T175815, start_date=20210306T175904, end_date=20210306T175904
[2021-03-06 17:59:04,727] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-06 17:59:04,766] {local_task_job.py:146} INFO - Task exited with return code 0
